lib/fluent/plugin/storage_local.rb

```diff
@@ -56,7 +56,7 @@ def configure(conf)
         elsif @path
           # ok
         else # @_plugin_id_configured is true
-          raise NotImplementedError, "implement this feature later with system_config"
+          raise NotImplementedError, "implement this feature later with system_config. Add <storage> section for file store or remove @id for memory store temporary"
```

Comment:
This block is for "Make root path for buffers and storages" (root path is based on `@id`) described in wiki. So it's wrong to tell users to remove `@id`.
https://github.com/fluent/fluentd/wiki/Changes-proposed-for-v0.14.x#multi-processes--cpu-cores